app/Http/Controllers/AuthController.php

```diff
@@ -90,6 +90,12 @@ public function logout(Request $request)
     public function user(Request $request)
     {
         try {
+            /*
+                TODO: Should be configurable?
```

Comment:
Imho, it's better to return the user, and we're already doing that. So the ones that dont need it can just ignore the returned user object.